### PR TITLE
Language/bengali

### PR DESCRIPTION
**Pull request by shovoncse**
Hello there,
I'm from **Bangladesh**
Please have a look at my **Bengali version** of 

> #StayTheFuckHome.

I've carefully done all the translations and others.
I love Your Movement to Stop the COVID-19 Pandemic.

Best Regards 
Shovon

### CODE REVIEW

**Review by sthakuri — APPROVED** [review]
Looks good. (I am not a Bengali native.)

**Review by webguru071 — APPROVED** [review]
I'm from Bangladesh and Bengali is my Native Language.
I think this translation is near perfect. 
It can be merged as it is.

Thanks

File: site/bn.njk [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12 font-bold">
+                <li class="pb-2">
+                    <h3>ফ্লুর চেয়ে আরও ছোঁয়াচে/h3>
+                    <p class="font-normal">
+                        1.4 - 6.49 এর মধ্যে একটি গড়  R0 অনুমান 3.28 <a class="source-link" href="https://academic.oup.com/jtm/advance-article/doi/10.1093/jtm/taaa021/5735319" target="_blank">[1]</a></sup> এর গড় অনুমানের সাথে, SARS-CoV-2 অনেক বেশি সংক্রামক এবং মৌসুমী ফ্লুর তুলনায় অনেক দ্রুত ছড়িয়ে পড়ে, যার একটি মধ্যম R0 রয়েছে 1.28<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/25186370" target="_blank">[2]</a></sup>।
+
+                        With an estimated R0 (the expected number of cases directly generated by one case in a population where all individuals are susceptible to infection) between 1.4 &ndash; 6.49 and a mean estimate of 3.28<sup>,
```
Comment (phelmig) [inline site/bn.njk]: Hi @shovoncse, the translation looks good so far, but seems to be incomplete at this point?

**Review by phelmig — CHANGES_REQUESTED** [review]

**Comment by faahim** [comment]
Hello @shovoncse ,

Great work Bhai! I also got interested and was so happy to see this PR already in place! Thanks for working on this! 💯 

The translation is great. However, I feel like it sounds a little too formal and it lacks the stong tone the original one has? I do understand that there's no proper synonym of the F-word in Bangla that doesn't sound a bit vulgar and that it might not be the most appropriate thing in some people's opinion. But I also think the strong phrasing of the original played an important role in getting the traction it's been getting?

So, I think I'd prefer to have a much stronger tone (without being vulgar, of course) to it. I believe that'll help to spread the message more easily and strongly. What are your thoughts on this?

**Comment by faahim** [comment]
Hi @phelmig ,

Since you're reviewing, I'd also like to know your thoughts on that?

Do you think we should try to preserve the strong tone of the original (the 'Stay the Fuck home' phrase for example)? 🙂 

Thanks!

File: site/bn.njk (hunk 1 above)
Comment (shovoncse) [reply site/bn.njk]: Hello @phelmig , Thank you for the suggestion. I didn't notice that point. Please check the current status. Thanks

**Comment by shovoncse** [comment]
Hello bro, @faahim 
Thanks for your Appreciation.
I had that problem with f-word when I was translating. I didn't find proper synonyms.
I also want to use a stronger tone to spread the message more easily and strongly. I think it will not be the same as English then. I've discussed it with my colleagues. However, I will love to see your suggestions on any point if you have any. 

Thanks in Advance

**Review by phelmig — APPROVED** [review]

**Pull request merged**